pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -820,6 +820,22 @@ func validateDetails(details jobspb.ChangefeedDetails) (jobspb.ChangefeedDetails
 		switch v := changefeedbase.FormatType(details.Opts[opt]); v {
 		case ``, changefeedbase.OptFormatJSON:
 			details.Opts[opt] = string(changefeedbase.OptFormatJSON)
+		case changefeedbase.OptFormatCSV:
+			initialScanType, err := initialScanTypeFromOpts(details.Opts)
+			if err != nil {
+				return jobspb.ChangefeedDetails{}, err
+			}
+			if initialScanType != changefeedbase.OnlyInitialScan {
+				return jobspb.ChangefeedDetails{}, errors.Errorf(
```

Comment:
I wonder why do you feel we have to have this restriction? is this purely because .. we don't know how to handle schema changes?

however cloud storage sinks name files with the "schema id" -- so changes in schema can be handled.

Reply:
Yeah this might be possible to address with cloud storage sinks, but what about non cloud storage sinks? When we address this issue for all of the sinks I would be more inclined to drop this restriction.